mlflow/pyspark/ml/__init__.py

```diff
@@ -256,6 +269,16 @@ def _get_param_map(instance):
     }
 
 
+def _get_params_param_map(instance):
```

Comment:
`_get_stage_type_params` looks like a better function name.
and add function doc.

Reply:
"stage" might not be accurate here because it includes "Evaluator" type instance here. But we already use "stage" name in other places. Let's use it for now

Reply:
The "Params" name is bad, it looks like some param instance but in fact it is the base class of Estimator/Transformer/Model/Evaluator. So using "Params" naming is very confusing.

Reply:
Done.